src/utils/ipfs/ipfs.d.ts

```diff
@@ -0,0 +1,31 @@
+import { IPFS, IPFSPath } from 'kubo-rpc-client/types';
```

Comment:
it is definetly not `utils`, maybe `/types`, or even combine files by entity (need discuss then)